barely_json/__init__.py

```diff
@@ -138,8 +136,8 @@ def decode_match(match):
           null | true | false | illegal)
 
 
-def default_resolver(value):
-    '''
+def default_resolver(value_):
```

Comment:
I will revert back this change and will create a separate PR for moving the grammar into a separate module.